articles/virtual-machines/extensions/update-linux-agent.md

```diff
@@ -61,7 +61,7 @@ AutoUpdate.Enabled=y
 To enable run:
 
 ```bash
-sudo sed -i 's/# AutoUpdate.Enabled=n/AutoUpdate.Enabled=y/g' /etc/waagent.conf
+sudo sed -i 's/# AutoUpdate.Enabled=y/AutoUpdate.Enabled=y/g' /etc/waagent.conf
```

Comment:
The original code is correct (it changes 'n' to 'y'), we should revert this change.

Reply:
Yes, but the file being changed does not contain 'n', it has the enabled state simply commented out. All **sed** needs to do is remove a comment.

This fails silently, since it succeeds by not making a change, and will go unnoticed by users who are following the docs.

Reply:
if it is commented out then it doesn't need any change. the default is 'y'. 

@MsGabsta - the doc should probably be updated. this needs to be changed only if the setting is uncommented and has a value of 'n'

Reply:
Thank you, I didn't notice that.